addons/sourcemod/scripting/jwp_skin.sp

```diff
@@ -315,28 +316,48 @@ bool IsVipSkinUse(int iClient)
 	return (VIP_IsClientVIP(iClient) && VIP_GetClientFeatureStatus(iClient, g_cVIPFeatureName) == ENABLED);
 }
 
-bool SetModel(int client, bool bSetDefaultModel = false)
+public Action SetModel(Handle timer, int client)
 {
-	if (!g_CvarTRandomSkins.BoolValue && !g_CvarCTRandomSkins.BoolValue) return false; // Disable it , if no random skins
+	bool bSetDefaultModel = false
```

Comment:
You're declared variable, that was a parameter to previous function, but it seems that you don't used it in code bottom.